sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -114,7 +114,7 @@ private[sql] class ParquetRelation2(
 
   // Should we merge schemas from all Parquet part-files?
   private val shouldMergeSchemas =
-    parameters.getOrElse(ParquetRelation2.MERGE_SCHEMA, "true").toBoolean
+    parameters.getOrElse(ParquetRelation2.MERGE_SCHEMA, sqlContext.getConf("spark.sql.parquet.mergeSchema" , "true") ).toBoolean
```

Comment:
Several styling issue here:

1. 100 columns exceeded
2. Remove the space before `,`
3. Remove the space before `)`